### ISSUE TITLE
Duplicate index ls_licensable_products_licensable_type_licensable_id_index

### ISSUE THREAD

**Issue by nagi1** [laravel-ready/license-server#1]
Hey, 

Fresh Laravel 9 installation
php 8.0.19
windows 10 x64

Reduce steps

- install laravel
- install the package
- don't publish the migrations
- migrate 

Screenshot:
![image](https://user-images.githubusercontent.com/16584220/174431565-ad0123db-0c81-4ff3-92b2-660e6959360f.png)

**Comment by nagi1** [comment]
Possible work around:

Change prefix in config file before publishing migrations and before `php artisan migrate` command.

in `license-server.php`

```php
    /**
     * This prefix will be used for all tables created by the license-server.
     *
     * Table name prefixes are used to prevent naming conflicts when using multiple stores
     * or local conflicts with other tables in the database.
     *
     * Example table name: ls_licenses
     *
     * Default: ls
     */
    'default_table_prefix' => 'your_project_name',
```

**Comment by relliv** [comment]
This issue was related to MySql's unique index query and running order. Fixed in [8151f1c](https://github.com/laravel-ready/license-server/commit/8151f1c18ed4417850ed7e0203ef6ccbc7bbdc16) commit.

Thanks for the feedback 🙌.
